tox.ini

```
@@ -1,76 +1,128 @@
 [tox]
-envlist = py{36,37,38}-{offline},online,build_docs,figure
-isolated_build = True
+envlist =
+    py{36,37,38}{,-oldestdeps,-devdeps,-online}
+    build_docs
+    figure
+    figure_astropydev
+    conda
```

```suggestion
    py36-conda
```
At least this is what the env is further down?

---

Fixed